lesson8/menu/menu.scss

```
@@ -0,0 +1,346 @@
+            top: 10%;
+            transform: translateY(-10%);
+            left: -22px;
+            border-width: 8px;
+            border-style: solid;
+            border-color: transparent $color-hover transparent transparent;
+            z-index: 99;
+        }
+
+        & .menu-bar__dropdown__women {
+            position: absolute;
+            top: 0;
+            left: 100%;
+            width: 80%;
+            background-color: $color-white;
+            transition: opacity 0.2s linear, height 0.2s linear;
+            border-left: 6px solid $color-hover;
+            z-index: 2;
+            opacity: 0;
+
+            & > .menu-bar__dropdown__item {
+                padding: 15px 20px;
+                border-bottom: 1px solid $color-border;
+                position: relative;
+
+                & > a {
+                    color: $color-gray;
+                    font-size: 20px;
+
+                    i {
+                        transition: transform 0.2s linear;
+                        margin-right: 15px;
+                    }
+
+                    & > div {
+                        display: inline-block;
+                        transition: transform 0.2s linear;
+                    }
+                }
+
+                &:hover > a{
+                    color: $color-hover;
+
+                    div {
+                        transform: translateX(10px);
+                    }
+
+                    i {
+                        transform: scale(1.2);
+                    }
+                }
+
+                &:hover .menu-bar__dropdown__about {
+                    display: block;
+                    opacity: 1;
+                    animation: fadeInDown 0.3s linear;
+                }
+
+                & .menu-bar__dropdown__about {
+                    position: absolute;
+                    top: 0;
+                    left: 100%;
+                    width: 80%;
+                    background-color: $color-white;
+                    transition: opacity 0.2s linear, height 0.2s linear;
+                    border-left: 6px solid $color-hover;
+                    z-index: 3;
+                    display: none;
+                    opacity: 0;
+
+                    & > li {
+                        padding: 15px 20px;
+                        border-bottom: 1px solid $color-border;
+                        position: relative;
+
+                        & > a {
+                            color: $color-gray;
+                            font-size: 20px;
+
+                            i {
+                                transition: transform 0.2s linear;
+                                margin-right: 15px;
+                            }
+
+                            & > div {
+                                display: inline-block;
+                                transition: transform 0.2s linear;
+                            }
+                        }
+
+                        &:hover > a{

```

```suggestion
                        &:hover > a {

```
Trước dấu `{` hãy thêm dấu cách